app/controllers/bookings_controller.rb

```diff
@@ -32,8 +32,16 @@ def update
     @booking = Booking.find(params[:id])
     @booking.update(booking_params)
```

Comment:
```suggestion
    # @booking.update(booking_params)
```
remove this line (we're updating the booking later on, on line 38, in our if/else statement)